.github/workflows/release.yml

```diff
@@ -0,0 +1,139 @@
+name: ezkl-lib-release
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  create-release:
+    name: create-release
+    runs-on: ubuntu-22.04
+
+    outputs:
+      upload_url: ${{ steps.create-release.outputs.upload_url}}
+      ezkl_version: ${{ env.EZKL_VERSION }}
+
+    steps:
+      - name: Get release version from tag
+        shell: bash
+        if: env.EZKL_VERSION == ''
+        run: |
+          echo "EZKL_VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          echo "version is: ${{ env.EZKL_VERSION }}"
+
+      - name: Create Github Release
+        id: create-release
+        uses: softprops/action-gh-release@v1
+        with:
+          token: ${{ secrets.RELEASE_TOKEN }}
```

Comment:
It should be possible to use `${{ github.token }}` otherwise I believe that a separate `RELEASE_TOKEN` will need to be set in the github secrets settings